lib/vagrant-scaleway/config.rb

```diff
@@ -17,6 +17,11 @@ class Config < Vagrant.plugin('2', :config)
       # @return [String]
       attr_accessor :image
 
+      # The image ID.
+      #
+      # @return [String]
```

Comment:
Not sure about this, I'm not very familiar with Ruby but noticed there is no Boolean class. What should I put?